### PR TITLE
Display infusions in gear and inventory when searching for infusions

### PR DESCRIPTION
**Pull request by n1tr0-5urf3r**

### CODE REVIEW

**Comment by Maselkov** [comment]
Could it maybe work for upgrades and runes too?

**Comment by n1tr0-5urf3r** [comment]
It does now

File: guildwars2/account.py [hunk 1]
```diff
@@ -318,19 +318,45 @@ def is_completed(encounter):
...
+        for character in characters:
+            char_names.append(character.name)
         for k, v in storage_counts.items():
             if v:
-                total += v
-                output.append("{} {} | {}".format(k.upper(),
+                if 'is_upgrade' in choice and choice['is_upgrade']:
+                    total += v
+                    if k in char_names:
+                        slotted_upg = await self.collect_upgrades(ctx, k.title(), choice["ids"][0])
```
Comment (Maselkov) [inline guildwars2/account.py]: That's going to be a lot of unnecesary API calls. Maybe we could add a new parameter to find_items_in_account that returns detailed results with per character equipped/not equipped?

**Comment by n1tr0-5urf3r** [comment]
I set a flag for find_items_in_account and create tuples of (item in inventory, item in gear) if it is set.
The flag defaults to false, so it won't break $li (again)
I'm then handling the tuple in $search command

**Pull request merged**